content/docs/start/experiments.md

```diff
@@ -46,10 +46,17 @@ The biggest difference to previous stages in our pipeline is in two new
 sections: `metrics` and `plots`. These are used to mark certain files containing
 experiment "telemetry". Metrics files contain simple numeric values (e.g. `AUC`)
 and plots files contain matrices and data series (e.g. `ROC` or model loss
-plots) that meant to be visualizing and compared.
+plots) that are meant to be visualizing and compared.
 
-> `cache: false` means that those file are small enough and versioned directly
-> with Git.
+The `-M` flag specifies a metrics file produced by this stage that is not
```

Comment:
we repeat about caching is three places now ... it's all very repetitive. We should rewrite it somehow. Flags - describe the primary meaning of them - metrics and plots, mention that they were used above in the `dvc run` ... etc

then cache: false, different types of flags, etc

Reply:
Continued in https://github.com/iterative/dvc.org/pull/1511#pullrequestreview-440251825. Resolving here.